src/app/[locale]/[channel]/p/[slug]/loading.tsx

```diff
@@ -0,0 +1,25 @@
+import { ProductImageGridSkeleton } from "@/app/[locale]/(components)/productImageGridSkeleton";
+import Badge from "@/app/daisyui/badge";
+
+export default function loading() {
+  return (
+    <main className="lg:flex lg:justify-between lg:mx-0 mt-4">
+      <section className="lg:w-2/3">
+        <ProductImageGridSkeleton />
+      </section>
+      <section className="lg:w-1/4 px-6 bg-almost-black glass rounded h-96 flex flex-col justify-start ml-2">
```

Comment:
I think this needs a little bit of padding on desktop, but otherwise this looks pretty good